code_to_gast/helpers/py_helpers.py

```diff
@@ -151,4 +158,34 @@ def name_to_gast(node):
 takes node of type unaryOp and converts it to our generic ast represenations
 """
 def unary_op_to_gast(node):
-    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
\ No newline at end of file
+    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
+
+"""
+takes node of type Compare and converts it to our generic ast representation
+example:
+    Example In: Compare(left=Num(n=1), ops=[Gt()], comparators=[Num(n=1), Num(n=2)])
+    Example Out: {'type': 'binOp', 'left': {'type': 'num', 'value': 1}, 'op': '>', 'right': {'type': 'num', 'value': 2}}
+"""
+def compare_to_gast(node):
+    # create list of comparators. (1>2>=3 would create list [1,2,3])
+    comparator_list = node.comparators
+    comparator_list.insert(0, node.left) # this is necessary since pyAST stores leftmost comparator seperately
+
+    # create list of operators. (1>2>=3 would create list [>, >=]
+    op_list = node.ops
+
+    return compare_helper(comparator_list, op_list)
+
+# TODO: combine logic in bool_op_helper and compare_helper into single function
```

Comment:
Is this for this PR or a future one?

Reply:
for a future PR. Since Cory and Steph are also working on stuff at the same time, I've been trying to avoid going in and making edits to code that's unrelated to the feature I'm adding.